### PR TITLE
Feature/protected gallery

### LINKED ISSUE #49: Add protected non user gallery

### PR DESCRIPTION
**Pull request by Segelzwerg**
closes #49

### CODE REVIEW

File: tests/routes/test_image_view.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from flask_api import status
+from lxml import html
```
[the remark this answers is not in the document]
Comment (Segelzwerg) [reply tests/routes/test_image_view.py:2]: I think it is still safe for us as it is used only for tests.

[the remark this answers is not in the document]
Comment (TomMD) [reply tests/routes/test_image_view.py:2]: Absolutely.  We're about to roll out an 'ignore file' feature with which we can exclude these results with a configuration file that shares .gitignore's syntax.

File: family_foto/__init__.py [hunk 2]
```diff
@@ -82,17 +83,17 @@ def create_app(test_config: dict[str, Any] = None, test_instance_path: str = Non
...
+    db.app = app
+    db.create_all()
 
     login_manager.init_app(app)
 
     from family_foto.web import photos, videos
     flask_uploads.configure_uploads(app, (photos, videos))
 
-    add_roles()
+    with app.app_context():
```
Comment (iTitus) [inline family_foto/__init__.py:95]: what does this do?

Comment (Segelzwerg) [reply family_foto/__init__.py:95]: It pushes the application into the call. This makes sure we can access the database. It should work without, but it was recommended here:
https://flask.palletsprojects.com/en/1.1.x/appcontext/#manually-push-a-context

And as this error occurred I added it.

**Review by iTitus — APPROVED** [review]

**Pull request merged**